### DISCUSSION TITLE
Deploy to Azure Button

### DISCUSSION THREAD

**Discussion by Smtuten** [microsoft/sample-app-aoai-chatGPT#675] [Q&A]
I deployed this sample app to my Azure tenant using the Deploy to Azure button.  The deployment was crazy easy and it works very well (thank you all for the hard work).  I don't think I'm losing my mind that the code has update, in my tenant, since I deployed it as the branding changed to Contoso.  To ask a dumb question, does my deployment get updated on some frequency or action with the main branch?  I ask because I would love to get the newest updates I see where you can do a little UI customization, such as change the title, logo, etc.  Can I pull the new code in an easy way without pushing the Deploy to Azure button to a new set of resources?  Thank you for any direction, as you can tell from  my post, this is not my strongest area.

**Comment by zedhaque** [comment]
You will have to fork/clone this repo for yourself. Make your changes and commit. 
Go to Azure portal app services under "overview" you will see that your current external git repo URL is this repo. You need to go to "Deployment Center" in "Azure App Services". Disconnect your current repo to your new repo and refresh. 

All future changes to this repo will need to manually pushed to your repo and then to your app.

**Reply by PeresDiego** [reply]
Hi @zedhaque  
I did exactly what you said, I have my brenches with the internal data, but when I point the app deployment to my brench there is an error.

If I use the deploy button, and create a new webapp it works perfectly but of the webapp container type, I would like to use the code type, to connect to my brench, can you help me?
